src/pages/design/icons/index.mdx

```diff
@@ -71,6 +71,11 @@ In certain instances where 16px icons will break the line height or will not fit
   </Do>
 </BestPractice>
 
+### Installation
+
+View the [zendeskgarden/svg-icons](https://github.com/zendeskgarden/svg-icons#garden-svg-icons---)
```

Comment:
Looks like there's three extra dashes at the end of the URL?

Reply:
Oh I see, it's from GitHub 👀

Reply:
I think https://github.com/zendeskgarden/svg-icons#readme is the preferred URL here.

Reply:
Haha no wonder I was like.. isn't there a better URL that points to the readme. I'll push up a PR to fix this.